### PR TITLE
[PT-BR] Add Bot account

### PR DESCRIPTION
**Pull request by daviirodrig**
- Add full pt-br translation of `Bot account` article

Note for future reviewers: I think `Conta de bot` is a better wording than `Conta de robô` or similar, since these days bot is a well know word in portuguese, also, on line 14, I don't know a good translation to `Multi-accounting` so I left it like this.

### CODE REVIEW

File: wiki/Bot_account/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+A principal diferença entre uma conta pessoal e contas de bot estão nos limites que se aplicam ao enviar mensagens no [chat](/wiki/Chat_Console). Contas de bot têm limites mais altos para permitir que seus serviços interajam com mais usuários sem risco de ter a conta [silenciada](/wiki/Glossary#silence):
+
+- Contas pessoais podem enviar 10 mensagens a cada 5 segundos
+- Contas de bot podem enviar 300 mensagens a cada 60 segundos.
+
+Estes limites se aplicam apenas para mensagens privadas, `#multiplayer`, e `#spectator`. Contas de bot não são permitidas de enviar mensagens em outros canais.
+
+## Criando uma conta de bot
+
+Se você está pensando em começar um chat bot, por favor inicie utilizando ele na sua conta do osu!, e certifique-se de respeitar os limites de conta pessoal listados acima. Assim que seu serviço crescer, pode ser uma oportunidade para solicitar uma conta de bot para o time de suporte de conta.
```
Comment (Flantsune) [inline wiki/Bot_account/pt-br.md:27]:
```suggestion
Se você está pensando em começar um chat bot, por favor inicie utilizando ele na sua conta do osu!, e certifique-se de respeitar os limites de conta pessoal listados acima. Assim que seu serviço crescer, pode ser uma oportunidade para solicitar uma conta de bot para a equipe de suporte de conta.
```
Just for consistency with other pt-br articles. If `equipe` is not deemed too good, we could change it to `time` in other pt-br articles if needed.

File: wiki/Bot_account/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+### Critérios mínimos
+
+Antes de considerar uma solicitação, o time de suporte requere que o bot cumpra estes critérios:
```
Comment (Flantsune) [inline wiki/Bot_account/pt-br.md:31]:
```suggestion
Antes de considerar uma solicitação, a equipe de suporte requere que o bot cumpra estes critérios:
```
Same thing as line 27.

File: wiki/Bot_account/pt-br.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- Um pequeno sumário do que seu bot faz
+
+O time de suporte de conta irá revisar sua solicitação. Se sua solicitação for negada, eles te dirão o porquê. Se sua solicitação for aprovada, eles criarão a conta de bot para você e te darão instruções sobre como usá-la.
+
+## Curiosidades
+
+- A insígnia do grupo `BOT` é mostrada em contas com um grupo primário de `Chat Bots` (grupo #29). A listagem deste grupo não é pública.
+- Algumas contas pessoais estão no grupo de chat bots pois seus donos não quiseram ou pediram para ter uma conta de bot separada para seus serviços, por exemplo ![][flag_DE] [Tillerino](https://osu.ppy.sh/users/2070907) e ![][flag_FR] [ThePooN](https://osu.ppy.sh/users/718454). Este tipo de instalação não é mais oferecido, e novos bots que entrarem no grupo sempre terão uma conta exclusiva.
+
+[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
```
Comment (Flantsune) [inline wiki/Bot_account/pt-br.md:61]:
```suggestion
[flag_DE]: /wiki/shared/flag/DE.gif "Alemanha"
```
This makes the pop-up when you hover the cursor read the following string.

For wiki reviewers: If needed, translation is in the pt-br meta info for country names. If this is unecessary please feel free to change back to English.

File: wiki/Bot_account/pt-br.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+[flag_FR]: /wiki/shared/flag/FR.gif "France"
```
Comment (Flantsune) [inline wiki/Bot_account/pt-br.md:62]:
```suggestion
[flag_FR]: /wiki/shared/flag/FR.gif "França"
```
Same as above.

**Comment by Flantsune** [comment]
In the case of `multi-accounting` most articles I have translated use the same term, I believe there is only one instance where `multi-contas` was used- And it's currently in an article that's being rewritten in English. It's okay to use that since the PT-BR community in general already knows and is used to that term.

**Comment by daviirodrig** [comment]
About `equipe` or `time`, they have literally the same meaning, so let's keep equipe for consistency as you said

**Pull request merged**